src/baselaplace.jl

```diff
@@ -30,11 +31,11 @@ function get_params(la::BaseLaplace)
     params = Flux.params(nn)
     n_elements = length(params)
     if la.subset_of_weights == :all
-        params = [θ for θ in params]                         # get all parameters and constants in logitbinarycrossentropy
+        params = [θ for θ in params]                                          # get all parameters and constants in logitbinarycrossentropy
     elseif la.subset_of_weights == :last_layer
-        params = [params[n_elements - 1], params[n_elements]]  # only get last parameters and constants
-    end
-    return params
+        params = [params[n_elements - 1], params[n_elements]]                   # only get last layer parameters and constants
+    end                                                                      # params[n_elements-1] is the weight matrix of the last layer
+    return params                                                            # params[n_elements] is the bias vector of the last layer
 end
```

Comment:
So the structure of `params` is like `[weights, biases, weights, biases, ...]`?

Reply:
As I understand it, yes.